pkg/sentry/kernel/BUILD

```diff
@@ -282,6 +282,7 @@ go_library(
         "//pkg/usermem",
         "//pkg/waiter",
         "@org_golang_x_sys//unix:go_default_library",
+        "@com_github_RoaringBitmap_roaring//:go_default_library",
```

Comment:
In general I think we try to avoid adding new dependencies to the sentry. See the section on "core" dependencies here:
https://github.com/google/gvisor/blob/master/CONTRIBUTING.md#coding-guidelines

Reply:
> In general I think we try to avoid adding new dependencies to the sentry. 

Hi Lan, thanks for informing me. Is it ok to use library `math/bits`? I can give it a try to use `math/bits` replace with `roaring bitmap`.

Reply:
Yeah, stdlib is fine.